src/webview/wkwebview/mod.rs

```diff
@@ -261,6 +261,9 @@ impl InnerWebView {
         let _: id = msg_send![_preference, setValue:_yes forKey:dev];
       }
 
+      #[cfg(target_os("macos"))]
+      let _: () = msg_send![_preference, setValue:_yes forKey:NSString::new("tabFocusesLinks")];
```

Comment:
should this be `let _: id = `?

Reply:
You are right with this one. But the syntax of cfg should be `#[cfg(target_os = "macos")]`

Reply:
Ah whoops, you're right